src/song_utils.py

```diff
@@ -162,7 +162,7 @@ def get_cover_art(filename: str) -> Optional[File]:
 
     # Make sure it doesn't go over 8MB
     # This is a safe lower bound on the Discord upload limit of 8MiB
```

Comment:
```suggestion
    # Make sure it doesn't go over the maximum size allowed for a Discord attachment.
    # Important for when the bot later posts the image during a bust.
```

I'd move the 8MB-specific text to the definition of `ATTACHMENT_BYTE_LIMIT`.